README.md

```diff
@@ -4,17 +4,17 @@
 For Advent Of Code 1 I choose From 2017 Day 2 <br>
 
 ### Solution
-~~~
+```kotlin
 var checkSum = 0
 val line = input.split('\n')
 for (values in line) {
     val num = values.split(' ')
     checkSum += num.max().toInt() - num.min().toInt()
 }
 println(checkSum)
-~~~
+```
 
-First I split the input by new Line and then loop through each values then i splited the values by space by which i extracted the min and max value using min() and max() function and converted it to integer and calculated the sum.
+First I split the input by new Line and then loop through each values then i splited the values by space by which i extracted the min and max value using `min()` and `max()` function and converted it to integer and calculated the sum.
```

Comment:
It's a good idea to wrap inline references to code in single backticks so that they get code highlighting in your README.